src/plugins/lightscript.js

```diff
@@ -143,7 +202,7 @@ pp.parseWhiteBlock = function (allowDirectives?, isExpression?) {
   // must start with colon or arrow
   if (isExpression) {
     this.expect(tt.colon);
-    if (!this.isLineBreak()) return this.parseMaybeAssign();
+    if (!this.isLineBreak()) return this.parseWhiteBlockBody(node, indentLevel, true, false);
```

Comment:
I will try that, I don't like `couldBeginStatement`. Not sure if there are any additional edge cases there but I can't think of any off hand.